geonode/layers/templates/layers/layer_leaflet_map.html

```diff
@@ -166,6 +151,7 @@
           onAdd: function(map) {
           var img = L.DomUtil.create('img');
 
+          // I'm not sure what is this logo supposed to be?
           img.src = '../static/img/logo.png';
```

Comment:
As a gut check, I would have thought that it should be `/static/geonode/img/logo.png` from `./geonode/static/geonode/img/logo.png`. I would personally play with the value and test to verify this, if I had time